scripts/update-spec.ts

```diff
@@ -1,252 +1,60 @@
 #!/usr/bin/env tsx
-import {readFile} from 'fs/promises'
+import { readFile } from 'fs/promises'
 import isEqual from 'lodash/isEqual'
-import mapValues from 'lodash/mapValues'
-import {load} from 'js-yaml'
-import {writeFile} from 'fs/promises'
-import glob from 'fast-glob'
+import { load } from 'js-yaml'
+import { Spec, baseSpecification } from './base-spec'
+import { writeFile } from 'fs/promises';
+import { fixDescriptionLinks, removeMultipleTagsFromEndpoints, mergeWithoutConflict } from './utils'
+import glob from 'fast-glob';
+
+
+// NOTE: hard coding these files again to avoid inclusion of all yaml files.
+// Using glob includes "enterprise-beta-apis.yaml" and "enterprise-audit-logs.yaml" which have duplicated apis that throw error
+const apis = [
+  './spec/public-api/platform.yaml',
+  './spec/public-api/platform-tags.yaml',
+  './spec/public-api/platform-experimental.yaml',
+  './spec/public-api/platform-containers.yaml',
+  './spec/public-api/platform-containers-experimental.yaml',
+  './spec/public-api/platform-items-bulk.yaml',
+  './spec/enterprise/enterprise-teams.yaml',
+  './spec/enterprise/enterprise-organizations.yaml',
+  './spec/enterprise/enterprise-board-classification.yaml',
+  './spec/enterprise/enterprise-board-export.yaml',
+  './spec/enterprise/enterprise-projects.yaml',
+];
+
+const baseYml = await glob("./spec/base.yaml");
+
+apis.push(baseYml[0]);
```

Comment:
Why do you need `glob` for one specific file?

```suggestion
apis.push('./spec/base.yaml');
```